internal/cephfs/clone.go

```diff
@@ -227,13 +228,13 @@ func (vo *volumeOptions) getCloneState(ctx context.Context, volID volumeID) (cep
 			vo.FsName,
 			string(volID),
 			err)
-		return cephFSCloneError, err
+		return cephFSCloneError, fmt.Errorf("failed to get FSAdmin: %w", err)
```

Comment:
Include some more details with every error, here you can use `"failed to get FSAdmin for volume %q: %w", vo, err`. I recommend to do something similar like this for other functions too.